spec/recipe_spec.rb

```diff
@@ -3,6 +3,7 @@
   ret = {'items' => 
   [
   {'day' => 1,
+   'value' => '{"id":655340,"imageType":"jpg","title":"Peanut Butter Swirl Banana Bread"}',
```

Comment:
redundant info but doesn't matter because this is fake data, only used to stub out api call while testing